tests/models/autoencoders/test_models_vae.py

```diff
@@ -361,9 +361,10 @@ class ConsistencyDecoderVAETests(ModelTesterMixin, unittest.TestCase):
     forward_requires_fresh_args = True
 
     def inputs_dict(self, seed=None):
-        generator = torch.Generator("cpu")
-        if seed is not None:
-            generator.manual_seed(0)
```

Comment:
This was likely a mistake because if a `seed` is provided you would want to use that to do: `generator.manual_seed(seed)` and not `generator.manual_seed(0)` using a fixed `seed`.